app/components/UI/Tabs/TabThumbnail/TabThumbnail.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useContext, useMemo } from 'react';
+import { Image, TouchableOpacity, View } from 'react-native';
+import ElevatedView from 'react-native-elevated-view';
+import { strings } from '../../../../../locales/i18n';
+import Avatar, {
+  AvatarSize,
+  AvatarVariant,
+} from '../../../../component-library/components/Avatars/Avatar';
+import Badge from '../../../../component-library/components/Badges/Badge/Badge';
+import { BadgeVariant } from '../../../../component-library/components/Badges/Badge/Badge.types';
+import BadgeWrapper from '../../../../component-library/components/Badges/BadgeWrapper';
+import Icon, {
+  IconColor,
+  IconName,
+  IconSize,
+} from '../../../../component-library/components/Icons/Icon';
+import Text, {
+  TextVariant,
+} from '../../../../component-library/components/Texts/Text';
+import AppConstants from '../../../../core/AppConstants';
+import { getHost } from '../../../../util/browser';
+import Device from '../../../../util/device';
+import { ThemeContext, mockTheme } from '../../../../util/theme';
+import WebsiteIcon from '../../WebsiteIcon';
+import createStyles from './TabThumbnail.styles';
+import { TabThumbnailProps } from './types';
+import useNetworkInfo from './useNetworkInfo';
+import useSelectedAccount from './useSelectedAccount';
+/* eslint-disable import/no-commonjs, @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports */
+const METAMASK_FOX = require('../../../../images/fox.png');
```

Comment:
I believe we can use import instead of require here
```suggestion
import METAMASK_FOX from '../../../../images/fox.png';
```

Reply:
thanks, this is now fixed, marking as resolved